src/components/PeoplePicker/PeoplePicker.Props.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { PeoplePicker } from './PeoplePicker';
+import { IPrincipal } from './Principal.Props';
+
+export interface IPeoplePickerProps extends React.Props<PeoplePicker> {
```

Comment:
Are all of these properties truly optional? Can the component work if one the properties is not provided?

Reply:
No, some props are not optional (onSearch, onSelect and suggestionList). I changed it